perfkitbenchmarker/providers/gcp/gce_network.py

```diff
@@ -245,58 +243,65 @@ def __init__(self, network_spec):
     self.project = network_spec.project
 
     #  Figuring out the type of network here.
-    #  Precedence: User Managed > CIDR (multi) > gce_subnet flag (single) > Auto (default)
-    self.net_type = 'default'  # Default network 'auto' mode. Single VPC with subnets in all regions
+    #  Precedence: User Managed > MULTI > SINGLE > DEFAULT
+    self.net_type = network.NetType.DEFAULT.value
     self.cidr = NETWORK_RANGE
-    if FLAGS.gce_subnet_region:  # Overrides auto network when set. Creates one VPC network in single region.
-      self.net_type = 'single'
+    if FLAGS.gce_subnet_region:
+      self.net_type = network.NetType.SINGLE.value
       self.cidr = FLAGS.gce_subnet_addr
-    if network_spec.cidr:  # Creates multiple VPC networks when set.
-      self.net_type = 'multi'
+    if network_spec.cidr:
+      self.net_type = network.NetType.MULTI.value
       self.cidr = network_spec.cidr
 
     name = self._MakeGceNetworkName()
 
-    subnet_region = FLAGS.gce_subnet_region if not network_spec.cidr else util.GetRegionFromZone(network_spec.zone)
+    subnet_region = FLAGS.gce_subnet_region if not network_spec.cidr else \
```

Comment:
\ is not glint compliant. Suggest using parenthesis instead.

Reply:
Fixed.
`grep -r --include \*.py -A 1 ' \\$'`  shows some other usages of this around the code that might need updated.